info about non-model artifacts (#5016)
iterative/dvc.org: content/docs/start/model-management/model-registry.md

Here the `path` parameter tells DVC that our model is to be found under
`"models/model.pkl"`, the `type` parameter is `"model"` and so it will show up

> in DVC Studio as a model type artifact (you can also set up DVC Studio to show
> other artifact types, but they are hidden by default). The rest of the

Reviewer: ```suggestion
in DVC Studio as a model type artifact (you can show
other artifact types using filters, but they are hidden by default). The rest of the
```

Just trying to clarify how to actually do this.
